src/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixer.php

```diff
@@ -232,7 +232,16 @@ private function prepareArgumentInformation(Tokens $tokens, int $start, int $end
         for ($index = $start; $index <= $end; ++$index) {
             $token = $tokens[$index];
 
-            if ($token->isComment() || $token->isWhitespace()) {
+            if (
+                $token->isComment()
+                || $token->isWhitespace()
+                || $token->isGivenKind([
+                    CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PRIVATE,
+                    CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PROTECTED,
+                    CT::T_CONSTRUCTOR_PROPERTY_PROMOTION_PUBLIC,
+                ])
+                || \defined('T_READONLY') && $token->isGivenKind(T_READONLY)
```

Comment:
```suggestion
                || (\defined('T_READONLY') && $token->isGivenKind(T_READONLY))
```
Cosmetic change for readability and safety (I prefer explicit groups over relying on and/or priority). But it's up to you, because it works like this.

Reply:
Are there any objective reasons to add such parentheses? For me, it decreases readability - when I see parentheses I assume they are for a reason.

Reply:
going with https://www.php.net/manual/en/language.operators.precedence.php

Reply:
Thanks @SpacePossum for quoting this, exactly my reasoning 👌.